theano/gpuarray/tests/test_opt.py

```diff
@@ -501,6 +501,8 @@ def test_no_complex():
 
 
 def test_local_lift_solve():
+    if not cusolver_available:
+        raise SkipTest('No cuSolver')
```

Comment:
This is good, but I think we should update the buildbot to use cuda8 too. @slefrancois how long would this take?